jablib/src/main/java/org/jabref/logic/citationstyle/CitationStyle.java

```diff
@@ -18,23 +18,25 @@ public class CitationStyle implements OOStyle {
     private final String title;
     private final boolean isNumericStyle;
     private final boolean hasBibliography;
+    private final boolean usesHangingIndent;
     private final String source;
     private final boolean isInternalStyle;
 
-    public CitationStyle(String filePath, String title, boolean isNumericStyle, boolean hasBibliography, String source, boolean isInternalStyle) {
+    public CitationStyle(String filePath, String title, boolean isNumericStyle, boolean hasBibliography, boolean usesHangingIndent, String source, boolean isInternalStyle) {
```

Comment:
We now have so many booleans that one could think about having an Enum and an EnumSet to pass all the options. But that can be done in a further PR